modules/react/layout/stories/stories_BoxVisualTesting.tsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 
 import {withSnapshotsEnabled} from '../../../../utils/storybook';
 import {Box} from '@workday/canvas-kit-react/layout';
-import {ComponentStatesTable} from '@workday/canvas-kit-labs-react/common';
 import {ContentDirection} from '@workday/canvas-kit-react/common';
```

Comment:
```suggestion
import {CanvasProvider, ContentDirection} from '@workday/canvas-kit-react/common';
```

This (and the change at the bottom) will fix your merge conflict with `prerelease/major`.